### PR TITLE
Embedded Object support.

### PR DESCRIPTION
**Pull request by steffenagger**
Embedded solution for beta.1 release.
This disables features allowing creation of embedded objects. Only listing & editing is supported in this 1st draft.
(`isEmbeddedType` should be seen as a temporary solution)

### CODE REVIEW

File: src/ui/RealmBrowser/index.tsx [hunk 1]
```diff
@@ -399,6 +403,26 @@ class RealmBrowserContainer
...
+      // TODO: Warning, not entirely sure about this one!
+      return !focus.property.isEmbedded;
+    }
+
+    return false;
+  };
+
+  private getFirstSchemaName = () =>
+    this.realm?.schema.find(c => c.name.indexOf('__') !== 0 && !c.embedded)
+      ?.name;
```
Comment (kraenhansen) [inline src/ui/RealmBrowser/index.tsx:424]: Should we move this to ui/RealmBrowser/schema-utils.ts?

File: src/ui/RealmBrowser/index.tsx [hunk 2]
```diff
@@ -706,9 +731,17 @@ class RealmBrowserContainer
     return Object.keys(objectSchema.properties).map(propertyName => {
       const property = objectSchema.properties[propertyName];
       if (typeof property === 'object') {
+        let isEmbedded = false;
+        if (property.objectType) {
+          const propertyObjectSchema = this.realm?.schema.find(schema => {
+            return schema.name === property.objectType;
+          });
+          isEmbedded = propertyObjectSchema?.embedded ?? false;
+        }
```
Comment (kraenhansen) [inline src/ui/RealmBrowser/index.tsx:740]: Consider simplifying this to two `const`s

**Review by kraenhansen — APPROVED** [review]
Great!

File: src/ui/RealmBrowser/index.tsx (hunk 1 above)
Comment (steffenagger) [reply src/ui/RealmBrowser/index.tsx]: It's not used anywhere else (anymore), so I'll move the check back in-line.

File: src/ui/RealmBrowser/index.tsx (hunk 2 above)
Comment (steffenagger) [reply src/ui/RealmBrowser/index.tsx]: Done... using `isEmbeddedType`.

**Review by kraenhansen — APPROVED** [review]

**Pull request merged**